### PR TITLE
Mahuika chours

### PR DESCRIPTION
**Pull request by ttgenius**
display mahuika along with maui core hours from 188 days ago (text and plot)

### CODE REVIEW

File: dashboard/run_data_collection.py [hunk 1]
```diff
@@ -47,6 +47,7 @@ class DataCollector:
 
     utc_time_format = "%m/%d/%y-%H:%M:%S"
     utc_time_gap = datetime.now() - datetime.utcnow()
+    total_start_time = "09/25/18-12:00:00"
```
Comment (claudio525) [inline dashboard/run_data_collection.py]: why this date? Doesn't our allocation start from beginning of june?

File: dashboard/run_data_collection.py [hunk 2]
```diff
@@ -258,7 +257,8 @@ def _parse_squeue(self, lines: Iterable[str]):
 
         return entries
 
-    def _parse_chours_usage(self, ch_lines: List):
+    @staticmethod
+    def parse_chours_usage(ch_lines: List):
```
Comment (claudio525) [inline dashboard/run_data_collection.py]: Why make this public?

File: dashboard/run_data_collection.py (hunk 1 above)
Comment (ttgenius) [reply dashboard/run_data_collection.py]: crop out the zeros

File: dashboard/run_data_collection.py (hunk 2 above)
Comment (ttgenius) [reply dashboard/run_data_collection.py]: it's used by run_old_data _colleciton

**Pull request merged**